docker-compose.yml

```diff
@@ -97,6 +97,20 @@ services:
       - serve
       - --enable-console
 
+  frontend:
+    container_name: eosrate_frontend
+    build:
+      context: ./services/frontend
+      dockerfile: ./Dockerfile
```

Comment:
@kuronosec we need to add a `target` parameter to build, so that we know we're only getting the small size stage image in the final build, meaning we need to add `target: server` 
https://docs.docker.com/compose/compose-file/
![image](https://user-images.githubusercontent.com/349542/78693504-95c12480-78b8-11ea-9045-3afa18fefa87.png)

Reply:
oh I see thanks :)